redis.cfg

```diff
@@ -33,12 +33,12 @@ redis-logfile = ${buildout:directory}/var/log/redis.log
 recipe = collective.recipe.scriptgen
 cmd = ${redis:location}/src/redis-server
 arguments =
-  ${buildout:directory}/redis.conf
+  ${buildout:bin-directory}/redis.conf
 
 [redis-conf]
 recipe = collective.recipe.template
-input = redis.conf.tpl
-output = redis.conf
+input = buildout-templates/redis.conf.tpl
+output = ${buildout:bin-directory}/redis.conf
```

Comment:
I'm generally sympathetic to that idea.  
e.g supervisor's recipe generates  parts/<part-name>/supervisord.conf,
I wouldn't be against any template recipe doing the same (e.g parts//<part-name>/<conf-file>. I *think* you can reference this in buildouts with ${<part-name>:location}/<conf-file>.